src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -226,6 +227,13 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ov::Model
     ADD_MATCHER(fq_fusions, MulFakeQuantizeFusion)
     fq_fusions->set_name("ov::pass::FakeQuantizeFusions");
 
+    // Temporary transformation to allow for PyTorch frontend to
+    // partially support bitwise operators with boolean inputs for plugins
+    // that didn't enabled BitwiseOps from opset13
+    auto bitwise = manager.register_pass<GraphRewrite>();
+    ADD_MATCHER(bitwise, ConvertBitwiseToLogical)
+    bitwise->set_name("ov::pass::ConvertBitwiseToLogical");
+
```

Comment:
I wonder, is the separate `bitwise` pass needed. It could be added to the `decomp` above (L147).
@itikhono @mateusztabaka Could you advise, what is a preffered way in this case?

Reply:
1. I think you don't need to create a separate GraphRewrite (line 233) here, ConvertBitwiseToLogical is already inherited from GraphRewrite in your case. It's better to use
`REGISTER_PASS(manager, ConvertBitwiseToLogical) `

2. You can add ConvertBitwiseToLogical to `decomp`, it's not an issue, I believe it will be even more optimized solution. But you need to make sure that you don't have conflicting transformations (e.g that matches exactly the same root node) registered inside one GraphRewrite.
 `decomp` adds all MatcherPasses from `ConvertBitwiseToLogical` to its own transformation list, so the effect is the same as if you just registered all MatcherPasses in `decomp` directly.

Reply:
Thanks, updated to use `REGISTER_PASS`